charts/ocis/ci/values.yaml

```diff
@@ -160,6 +161,8 @@ services:
       requests:
         cpu: 100m
         memory: 128Mi
+    jobNodeSelector:
+      kubernetes.io/arch: "arm64"
```

Comment:
Was it intended to use `arm64` instead of `amd64` here?

Reply:
I think it could be desired because if you check the templated Kubernetes resources you'll see a difference
If you wouldn't see a difference you'd see that there is a bug
Eg that were using the regular nodeSelector instead of the jobNodeSelector